src/tests/jsonLogic.test.js

```diff
@@ -363,58 +371,205 @@ describe('jsonLogic: cross-values validations', () => {
       expect(handleValidation({ field_a: 10 }).formErrors).toEqual(undefined);
       expect(fieldB.label).toEqual('I need this to work using the 20.');
     });
-  });
 
-  it('Use multiple inline rules with different identifiers', () => {
-    const { fields, handleValidation } = createHeadlessForm(
-      schemaWithInlineMultipleRulesForComputedAttributes,
-      {
+    it('Use multiple inline rules with different identifiers', () => {
+      const { fields, handleValidation } = createHeadlessForm(
+        schemaWithInlineMultipleRulesForComputedAttributes,
+        {
+          strictInputType: false,
+        }
+      );
+      const [, fieldB] = fields;
+      expect(handleValidation({ field_a: 10, field_b: null }).formErrors).toEqual(undefined);
+      expect(fieldB.description).toEqual('Must be between 5 and 20.');
+    });
+
+    it('Use an inline rule in a schema for a title but it just uses the value', () => {
+      const { fields, handleValidation } = createHeadlessForm(schemaInlineComputedAttrForTitle, {
         strictInputType: false,
-      }
-    );
-    const [, fieldB] = fields;
-    expect(handleValidation({ field_a: 10, field_b: null }).formErrors).toEqual(undefined);
-    expect(fieldB.description).toEqual('Must be between 5 and 20.');
-  });
+      });
+      const [, fieldB] = fields;
+      expect(handleValidation({ field_a: 10, field_b: null }).formErrors).toEqual(undefined);
+      expect(fieldB.label).toEqual('20');
+    });
+
+    it('Use an inline rule for a minimum, maximum value', () => {
+      const { fields, handleValidation } = createHeadlessForm(
+        schemaInlineComputedAttrForMaximumMinimumValues,
+        {
+          strictInputType: false,
+        }
+      );
+      const [, fieldB] = fields;
+      expect(fieldB).toMatchObject({ minimum: NaN, maximum: NaN });
```

Comment:
@sandrina-p ... can you please review this test specifically based off our convo https://github.com/remoteoss/json-schema-form/pull/40#discussion_r1311418085

is this right for the initial value?

Reply:
Even after reading the comments and code, I'm not 100% sure on this one. We can proceed and then come back and adjust if needed?

Reply:
Looking at this from the outside, `NaN` doesn't make sense. When the json-logic is incomplete (aka missing a value for the calculation), I'd rather not run it and just return `undefined`, as if it didn't exist. WDYT?

Since this is an inoffensive edge-case (I hope!) and don't want to slow you down, I'd just leave a comment explaining why it's NaN and the linear issue ID for us to follow-up later on.